### PR TITLE
Add %U format specifier (unread articles count) for feedlist-title-format

### PR DESCRIPTION
**Pull request by dennisschagt**
Resolves https://github.com/newsboat/newsboat/issues/1495.

### CODE REVIEW

File: doc/newsboat.asciidoc [hunk 1]
```diff
@@ -1168,6 +1168,7 @@ Identifier:Meaning
 |======================================================================
 Identifier:Meaning
 [[feedlist-title-format-T]]<<feedlist-title-format-T,+T+>>:Currently selected tag (empty if none selected)
+[[feedlist-title-format-U]]<<feedlist-title-format-U,+U+>>:Number of unread articles
```
Comment (Minoru) [inline doc/newsboat.asciidoc:1171]: The description for `%u` above also says "number of unread articles", so now the tables are confusing. Can you please move `%u` (and also `%t` and maybe `%F`) out of the above table and into this and the next one, giving them accurate descriptions so there is no confusion over what they actually mean?

**Review by Minoru — APPROVED** [review]
Thanks! I think this can be merged once the docs are polished a bit (see below).

File: doc/newsboat.asciidoc (hunk 1 above)
Comment (dennisschagt) [reply doc/newsboat.asciidoc:1170]: Good idea, done: https://github.com/newsboat/newsboat/pull/1496/commits/9f93fcd5f33a9fb4e3c881fcba6cd660ced6817a

**Comment by Minoru** [comment]
Thanks! Let's see how auto-merge works :) I tried it on a different PR yesterday, and it simply hanged with a message "Trying to auto-merge". But I was changing repo settings at the time, maybe that threw it off.

**Comment by Minoru** [comment]
It worked \o/

**Pull request merged**